DESCRIPTION

```diff
@@ -78,16 +64,14 @@ Suggests:
     waldo,
     xml2
 LinkingTo: 
-    progress (>= 1.2.1),
     cpp11 (>= 0.2.0),
+    progress (>= 1.2.1),
     tzdb (>= 0.1.1)
 VignetteBuilder: 
     knitr
+Config/Needs/website:nycflights13, tidyverse/tidytemplate
 Config/testthat/edition: 3
 Config/testthat/parallel: false
-Config/Needs/website:
-    nycflights13,
-    tidyverse/tidytemplate
```

Comment:
I would accept the new **location** of `Config/Needs/website`, but not the re-formatting. This will eventually get fixed in desc, but in the meantime, just retain the preferred formatting "by hand".

For background:
https://github.com/r-lib/desc/issues/117
https://github.com/r-lib/desc/issues/122